### PR TITLE
V9 signal samples

### PR DESCRIPTION
**Pull request by kmohrman**
This PR updates all of the JSONS for the signal samples to point to our newly-produced v9 signal samples. This would close Issue #87. Also included in the PR:
* Overwriting the sum of weights in the JSONs to be the correct sum of weights (using the sum of weights processor and `update_json_sow.py` script.
* A couple minor typo fixes for the sum of weights and update json tools.
* Replacing the ROOT file we use for the CI to be one of the new v9 files. 
* Remaking all of the reference files (for datacards and yield JSONs).

### CODE REVIEW

**Comment by kmohrman** [comment]
Hi @bryates and @Andrew42, just wanted to check in about the status of this PR. I'm wondering if there are any issues or items that should be addressed? If you need more time to review that's of course ok too, we just have a lot of open PRs so it'd probably be good to start moving forward on closing some of them if we can.

File: analysis/topEFT/remake_ci_ref_yields.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 # Get the file the CI uses, and move it to the directory the JSON expects
 printf "\nDownloading root file...\n"
-wget -nc http://www.crc.nd.edu/~ywan2/root_files/NAOD-00000_18449.root ../../NAOD-00000_18449.root
+wget http://www.crc.nd.edu/~kmohrman/files/root_files/for_ci/ttHJet_UL17_R1B14_NAOD-00000_10194.root
```
Comment (bryates) [inline analysis/topEFT/remake_ci_ref_yields.sh:6]: We should keep the `-nc` flag to prevent redownloading if the file already exists locally.

Comment (kmohrman) [reply analysis/topEFT/remake_ci_ref_yields.sh]: Thanks for catching that @bryates, I guess I accidentally removed that flag when pasting in the path to the new file. I've added the `-nc` back in now.

Comment (bryates) [reply analysis/topEFT/remake_ci_ref_yields.sh]: No worries, thanks for the fix.

**Review by Andrew42 — APPROVED** [review]
As this PR basically just updates the json files to point to the new samples, I don't see any major issues with the changes here. You've made sure to re-run the post-sample processing stuff like extracting the appropriate sum of weights and inserting it into the json files. You've also made sure to update and replace the relevant reference files used for our CI/CD pipeline.

I'd say the most important thing is that we didn't forget to update one the associated files or have an outdated file pointing to the old samples lying around somewhere. If we do find out some deeper issue with the sample themselves we can always handle that with a separate PR (which would help to keep things more focused anyways). So I'm fine with this PR as is.

**Comment by kmohrman** [comment]
> As this PR basically just updates the json files to point to the new samples, I don't see any major issues with the changes here. You've made sure to re-run the post-sample processing stuff like extracting the appropriate sum of weights and inserting it into the json files. You've also made sure to update and replace the relevant reference files used for our CI/CD pipeline.
> 
> I'd say the most important thing is that we didn't forget to update one the associated files or have an outdated file pointing to the old samples lying around somewhere. If we do find out some deeper issue with the sample themselves we can always handle that with a separate PR (which would help to keep things more focused anyways). So I'm fine with this PR as is.

Thanks for the review @Andrew42. @bryates, did you have any other comments or were there any other things you wanted to check, or should I go ahead and merge now?

**Comment by bryates** [comment]
> > As this PR basically just updates the json files to point to the new samples, I don't see any major issues with the changes here. You've made sure to re-run the post-sample processing stuff like extracting the appropriate sum of weights and inserting it into the json files. You've also made sure to update and replace the relevant reference files used for our CI/CD pipeline.
> > I'd say the most important thing is that we didn't forget to update one the associated files or have an outdated file pointing to the old samples lying around somewhere. If we do find out some deeper issue with the sample themselves we can always handle that with a separate PR (which would help to keep things more focused anyways). So I'm fine with this PR as is.
> 
> Thanks for the review @Andrew42. @bryates, did you have any other comments or were there any other things you wanted to check, or should I go ahead and merge now?

I don't have anything to add, but I never did test it on my side. If everything works for you, then I'm happy with merging now.

**Comment by kmohrman** [comment]
Just for future reference (at least as long as the link exists anyway), here is the [link](http://www.crc.nd.edu/~kmohrman/EFT/TopCoffea/test_naod_samples/validate_v9_eft_samples/dec21_test_master_vs_v9_noPrefireWeights/) to the v8 vs v9 comparison plots I made on this branch last week (essentially the change is very small from v8 to v9).

**Pull request merged**